docs/tutorials/t2Ramsey_tutorial.ipynb

```diff
@@ -0,0 +1,415 @@
+{
```

Comment:
Is it intentional that there are two notebooks? Possibly `t2ramsey_tutorial` with small `r` is better.

Reply:
Strange, I don't see a `t2Ramsey_tutorial.ipynb` with capital `R`. I did find a `t2star_tutorial.ipynb` and removed it.